core/src/main/scala/kafka/server/BrokerMetadataCheckpoint.scala

```diff
@@ -88,9 +88,9 @@ class RawMetaProperties(val props: Properties = new Properties()) {
   }
 
   override def toString: String = {
-    "RawMetaProperties(" + props.keySet().asScala.toList.asInstanceOf[List[String]].sorted.map {
+    "{" + props.keySet().asScala.toList.asInstanceOf[List[String]].sorted.map {
```

Comment:
Is this producing JSON? If so, should we use a JSON encoder instead?

Reply:
It doesn't produce JSON (for example, keys and values are not quoted).

This is intended for human consumption.  It will show up in the command-line output.

Reply:
ok, sounds good